core-bundle/src/String/SimpleTokenParser.php

```diff
@@ -168,8 +168,6 @@ private function getVariables(string $expression): array
 
             // Skip functions
             if (isset($tokens[$i + 1]) && '(' === $tokens[$i + 1]->value) {
-                ++$i;
-
```

Comment:
This change is not functionally equivalent anymore, because now the next iteration *will* be executed, while it would be skipped before.

Reply:
It does not seem to make a difference though. Or maybe the unit tests do not cover it completely?

Reply:
I have reverted the change.